test/integration/scenario.ts

```diff
@@ -310,50 +288,30 @@ describe('Scenario test multi-pair', function () {
           ldo = await ethers.getContractAt('IERC20', mainnet.LDO)
           await ldo.connect(agent).transfer(await stonks.getAddress(), value)
 
-          expect(isClose(await ldo.balanceOf(await stonks.getAddress()), value))
+          expect(isClose(await ldo.balanceOf(await stonks.getAddress()), value, 1n))
         })
         it('manager should recover unexpected token', async () => {
           const agentBalanceBefore = await ldo.balanceOf(mainnet.AGENT)
           await stonks
             .connect(manager)
-            .recoverERC20(
-              await ldo.getAddress(),
-              await ldo.balanceOf(await stonks.getAddress())
-            )
-          expect(
-            isClose(await ldo.balanceOf(await stonks.getAddress()), BigInt(0))
-          )
-          expect(
-            isClose(
-              await ldo.balanceOf(mainnet.AGENT),
-              agentBalanceBefore + value
-            )
-          )
+            .recoverERC20(await ldo.getAddress(), await ldo.balanceOf(await stonks.getAddress()))
+          expect(isClose(await ldo.balanceOf(await stonks.getAddress()), BigInt(0), 1n))
```

Comment:
LDO has no rounding errors on transfers, so it may be used strict 
equality checks